packages/atomic/src/components/common/checkbox.ts

```diff
@@ -0,0 +1,72 @@
+import {html, TemplateResult} from 'lit';
+import {classMap} from 'lit/directives/class-map.js';
+import {ifDefined} from 'lit/directives/if-defined.js';
+import {ref, RefOrCallback} from 'lit/directives/ref.js';
+import Tick from '../../images/checkbox.svg';
+
+export interface CheckboxProps {
+  checked: boolean;
```

Comment:
I believe Lit follows the HTML specification for boolean attributes, i.e., if it is specified, it evaluates to `true` (such that, unlike with Stencil, `checked="false"` would actually evaluate to `true`).

Givn that, should we just make the `checked` prop optional in the Lit component?

Reply:
makes sense